README.md

```diff
@@ -11,26 +11,39 @@ This project is a moblie website for personal portfolio. Home page of this proje
 Major languages:- HTML, CSS
 Technologies:- VScode
 
-## Getting Started
+##Project Link:- https://jusha25.github.io/personal-portfolio/
 
-To view this project, download the repo from github and run on the local machine browser.
+##Getting Started:-
+
+To get a local copy up and running follow these simple example steps.
+
+##Visit And Open Files
+https://github.com/Jusha25/personal-portfolio
+
+##Download Repo
+run command:- git clone https://github.com/Jusha25/personal-portfolio.git
```

Comment:
- [ ] As suggested by the previous code reviewer, please add steps to run the project locally:

```
## Getting Started
To get a local copy up and running, follow the steps in Setup below.

### Setup
If you installed git you can clone the code to your machine, or download a ZIP of all the files directly.

[Download the ZIP from this location](https://github.com/Jusha25/personal-portfolio/archive/refs/heads/master.zip), or run the following [git](https://git-scm.com/downloads) command to clone the files to your machine:

```bash
git clone https://github.com/Jusha25/personal-portfolio.git
```

```
- Once the files are on your machine, open the _personal-portfolio_ folder in [Visual Studio Code](https://code.visualstudio.com/)

- With the files open in Visual Studio Code, press the **Go Live** button at the bottom of the window to launch the files with [Live Server](https://marketplace.visualstudio.com/items?itemName=ritwickdey.LiveServer).
```